catalogs/README.rst

```diff
@@ -6,12 +6,23 @@ This means we don't rely on other websites and data sources to be stable.
 
 The licences and references are:
 
-- 1LHAASO:
+- The first LHAASO catalog (1LHAASO):
     - Reference: `Zhen Cao et al. 2024 ApJS 271 25 <https://iopscience.iop.org/article/10.3847/1538-4365/acfd29>`_
     - Licence: The First LHAASO Catalog of Gamma-Ray Sources © 2023 by The LHAASO
       Collaboration is licensed under `CC BY-NC-SA 4.0 <https://creativecommons.org/licenses/by-nc-sa/4.0/>`_.
-- 2HWC:
-- 3HWC:
-- HGPS:
+- The Second HAWC Catalog of Very-high-energy Gamma-Ray Sources (2HWC):
+    - Reference: `A. U. Abeysekara et al., 2017, ApJ 843 40 <https://iopscience.iop.org/article/10.3847/1538-4357/aa7556>`_
+    - Licence: adapted and modified from the HAWC public datasets (see `this link <https://data.hawc-observatory.org/about.php>`_),
+      these data are licenced like the other Gammapy data, ie under
+      `CC BY-NC-SA 4.0 <https://creativecommons.org/licenses/by-nc-sa/4.0/>`_.
+- The Third HAWC Catalog of Very-high-energy Gamma-Ray Sources (3HWC):
+    - Reference: `A. Albert et al., 2020, ApJ 905 76 <https://iopscience.iop.org/article/10.3847/1538-4357/abc2d8>`_
+    - Licence: adapted and modified from the HAWC public datasets (see `this link <https://data.hawc-observatory.org/about.php>`_),
```

Comment:
Is this correct? The catalog is gammapy Data is not 
" adapted and modified from the HAWC public datasets", but directly the 3HWC catalog.

Reply:
The HAWC catalog that we have has been produced by us with a given format. The used python codes are within the directory (e.g. make_2hawc.py). The 'data' have been then modified. This is why one can not use directly the HAWC licence, but rather our licence.